packages/create-vite/src/index.ts

```diff
@@ -267,17 +286,24 @@ async function init() {
         },
         {
           type: () =>
-            !fs.existsSync(targetDir) || isEmpty(targetDir) ? null : 'confirm',
+            !fs.existsSync(targetDir) || isEmpty(targetDir) ? null : 'select',
           name: 'overwrite',
           message: () =>
             (targetDir === '.'
               ? 'Current directory'
               : `Target directory "${targetDir}"`) +
-            ` is not empty. Remove existing files and continue?`,
+            ` is not empty. Please choose how to proceed:`,
+          initial: 0,
+          choices: OVERWRITEOPTIONS.map((overwriteOption) => {
+            return {
+              title: overwriteOption.display,
+              value: overwriteOption.value,
+            }
+          }),
```

Comment:
Since we only reference `OVERWRITEOPTIONS` here, I think we can inline the array and pass it to `choices` directly. So something like:

```js
choices: [
  {
    title: 'Remove existing files and continue',
...
```